routes/movies.routes.js

```diff
@@ -0,0 +1,100 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require('express').Router();
+
+const Movie = require('../models/Movie.model');
+const Celebrity = require('../models/Celebrity.model');
+
+// all your routes here
+
+// show list of movies
+
+router.get('/movies', (req, res, next) => {
+	Movie.find()
+		.then((moviesFromDB) => {
```

Comment:
I would say to keep the same style for the different routes, at least in the project, possible keep one or the other